docs/release-notes/may-2024.md

```diff
@@ -0,0 +1,140 @@
+---
+slug: May-2024
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: May 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for May 2024
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+- **Semgrep managed scanning** is now in public beta. It enables you to scan with Semgrep on any GitHub-hosted repository without the need to change existing CI/CD configurations.
+  - Managed scanning is crucial to organizations with hundreds or thousands of repositories. Read the [<i class="fa-regular fa-file-lines"></i> docs](/deployment/managed-scanning) or the [<i class="fas fa-external-link fa-xs"></i> blog post](https://semgrep.dev/blog/2024/rapidly-deploy-code-scans-with-semgrep-managed-scanning).
+
+### Changed
+
+- Improved the onboarding experience when scanning with Semgrep on a remote repository.
+- **Projects page**: Improved the **Can't find your project?** tooltip with more troubleshooting advice.
+  - The **Sync projects** button has been moved to the **Can't find your project** tooltip.
+- Improved Jira integration and workflow. Jira tickets can now be created for Supply Chain findings, and provide better ticket summaries <!-- 14334 --> and descriptions. <!-- 14253 -->
+  - The Jira integration is in a private beta. See the [<i class="fa-regular fa-file-lines"></i> Jira documentation](/semgrep-appsec-platform/jira) to learn how it works and how to gain access to the beta.
+- **Editor and Playground**: various improvements to structure mode, including the addition of tooltips to aid in rule writing and bug fixes.
+
+## 💻 Semgrep Code
+
+### Added
+
+- **Code search** is now in beta. Code search allows you to test a Semgrep rule by running it against one or more GitHub repositories or projects instead of just a few lines of test code. Its results highlight all instances of matching code in those target repositories, allowing you to see whether your rule works as intended or not.
+- **Pro findings** filter: when reviewing items on Semgrep AppSec Platform's **Findings** page, the **Pro findings** filter allows you to filter for findings identified using Semgrep Pro rules, as well as findings originating from Pro Engine analysis; if Semgrep identifies a finding as the result of Pro Engine analysis, or interfile and interprocedural analysis, it's included by this filter.
```

Comment:
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/6543f675-db3b-4b52-9178-9c87338fd5b1)
 
Minor suggestion. This entire chunk is one sentence. Maybe break up?

Reply:
Side note: should we always be saying _The_ Semgrep AppSec Platform? To me it reads oddly when we omit "the" because it's technically grammatically incorrect

Reply:
Our internal styleguide of sorts called for the omission of "the" when referring to SCP so I've carried that practice over to SAP; unfortunately I can't find the discussion where that was decided on

Reply:
It's this: https://www.notion.so/semgrep/Semgrep-technical-communication-style-guide-4d1104e78927460aa8c7624aedd6561f?pvs=4#64ffc68b6123419aa3bfa444a1f26461

There was actually a much longer discussion about how Semgrep Cloud Platform (the old name) "looks" wrong because "Platform" reads as a common noun, even if Semgrep AppSec Platform as a whole **is a proper noun**.

For example:

- Dave is obviously a proper noun so we say `Dave detected a finding`, not `The Dave detected a finding`.
- Semgrep Secrets is a proper noun, **and it's still obviously a proper noun** so `The Semgrep Secrets detected a finding` is "obviously" wrong.
- Now what if the name of our product was Acorn? It all of a sudden sounds more natural to read `The acorn detected a finding` - but in this context, Acorn is a proper noun and it should be `Acorn detected a finding`.

Reply:
This makes total sense!